build.sbt

```diff
@@ -44,14 +45,23 @@ lazy val `sbt-plugin` = projectMatrix
   .dependsOn(core)
   .enablePlugins(ScriptedPlugin)
   .settings(
-    name := """sbt-vcpkg""",
```

Comment:
sbt plugins are usually named `sbt-xyz`. Why change it?

Reply:
Note that this is still a draft PR and is constantly changing :)

Reply:
In the end I think it will be `sbt-vcpkg` and `mill-vcpkg`.

Reply:
It was to keep consistency with the `vcpkg-core` module, but I'll abide with whatever @keynmol says

Reply:
Let's go with 

- `core`
- `sbt-vcpkg`
- `mill-vcpkg`

It seems to follow what ScalablyTyped is like and I want to mimic it.
Also, can be changed before the release, no biggie.

Reply:
no problem, I'll amend it here :)